docs/FeatureManagement/Samples/NoFilters.tests.json

```diff
@@ -0,0 +1,42 @@
+{
+    "1": {
```

Comment:
The index number is hard to maintain. Why is it necessary? Can we just use array? Everything in an array is automatically numbered.

Reply:
I was wanting an ID field so could discuss an individual test, as we can't use the feature flag name to do that as it is many to one. We could change it to be something like `NoFilters1` or something like that. And/or we could move the value into the object.